eth/fetcher/tx_fetcher_test.go

```diff
@@ -1125,8 +1125,8 @@ func TestTransactionFetcherBandwidthLimiting(t *testing.T) {
 						{common.Hash{0x06}, types.LegacyTxType, maxTxRetrievalSize},
 					},
 					"C": {
-						{common.Hash{0x07}, types.BlobTxType, params.MaxBlobGasPerBlock},
-						{common.Hash{0x08}, types.BlobTxType, params.MaxBlobGasPerBlock},
+						{common.Hash{0x07}, types.BlobTxType, params.BlobTxBlobGasPerBlob * 10},
+						{common.Hash{0x08}, types.BlobTxType, params.BlobTxBlobGasPerBlob * 10},
```

Comment:
What if the maximum is increased in the future? These numbers won't be accurate, and unless someone remembers to change that here, it won't be representative of what the max could be. I would suggest keeping some max value in `params`, that is used in the Prague config, and here as well.

Reply:
It's just a simple data-passing test. The same value is sent a few lines up in `doTxNotify`.